haptics/ios/Plugin/Haptics.swift

```diff
@@ -0,0 +1,71 @@
+import AudioToolbox
+import UIKit
+
+@objc public class Haptics: NSObject {
+
+    var selectionFeedbackGenerator: UISelectionFeedbackGenerator?
+
+    @objc public func impact(_ style: String?) {
+        DispatchQueue.main.async {
+            if let style = style {
+                var impactStyle = UIImpactFeedbackGenerator.FeedbackStyle.heavy
+                if style == "MEDIUM" {
+                    impactStyle = UIImpactFeedbackGenerator.FeedbackStyle.medium
+                } else if style == "LIGHT" {
+                    impactStyle = UIImpactFeedbackGenerator.FeedbackStyle.light
+                }
```

Comment:
I think it would be better to move this string comparison to the Plugin class. Then this method could take a `FeedbackStyle` parameter. That would make this class far more usable by other code instead of just taking an undocumented `String` parameter where the expected values are unclear.

Reply:
Then I don't think we need the Haptics.swift class at all, users can just use the Haptics classes themselves